### PR TITLE
Update detector model used for CI and add tests for LCFIJet and `--help`

### PR DESCRIPTION
**Pull request by Zehvogel**
BEGINRELEASENOTES
- Updated detector model used for CI to `CLD_o2_v06` and added tests for LCFIJet and `k4run --help`

ENDRELEASENOTES

### CODE REVIEW

File: CLDConfig/CLDReconstruction.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
 from Configurables import GeoSvc, TrackingCellIDEncodingSvc
 geoservice = GeoSvc("GeoSvc")
-geoservice.detectors = [os.environ["K4GEO"]+"/FCCee/CLD/compact/CLD_o2_v05/CLD_o2_v05.xml"]
+geoservice.detectors = [os.environ["K4GEO"]+"/FCCee/CLD/compact/CLD_o2_v06/CLD_o2_v06.xml"]
```
Comment (andresailer) [inline CLDConfig/CLDReconstruction.py:58]: We need to update the default simulation model as well in this case https://fcc-ee-detector-full-sim.docs.cern.ch/CLD/

Comment (Zehvogel) [reply CLDConfig/CLDReconstruction.py:58]: Yes we also need to update that.

Is it possible to add a default model also to `cld_steer.py`?

Comment (Zehvogel) [reply CLDConfig/CLDReconstruction.py:58]: apparently yes :D

I will update the documentation to also use the steering file

Comment (Zehvogel) [reply CLDConfig/CLDReconstruction.py:58]: But I will also update the model in the doc explicitly and then we can drop it from the example after the next release

Comment (Zehvogel) [reply CLDConfig/CLDReconstruction.py:58]: https://gitlab.cern.ch/fcc-ee-detector-full-sim/docs/-/merge_requests/10

**Comment by Zehvogel** [comment]
We should wait for https://github.com/key4hep/key4hep-spack/issues/605

File: CLDConfig/CLDReconstruction.py (hunk 1 above)
Comment (andresailer) [reply CLDConfig/CLDReconstruction.py:58]: thanks!

Comment (Zehvogel) [reply CLDConfig/CLDReconstruction.py:58]: Documentation MR is merged

**Pull request merged**